chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -295,6 +304,7 @@ impl PeerManagerActor {
         client_addr: Recipient<NetworkClientMessages>,
         view_client_addr: Recipient<NetworkViewClientMessages>,
         routing_table_addr: Addr<RoutingTableActor>,
+        ping_counter: Box<dyn PingCounter>,
```

Comment:
Rather than adding a new parameter and adjusting all call-sites, consider using builder-lite pattern:

```
pub fn with_ping_counter(self, c: Box<dyn Counter>) -> Self
```

(main just uses default (), Box<()> won’t even allocate)

Reply:
done